src/beast/beast/utility/Journal.h

```diff
@@ -54,11 +43,30 @@ class Journal
         kDisabled,
         kNone = kDisabled
     };
+}
+
+/** A generic endpoint for log messages.
+
+    The Journal has a few simple goals:
+
+     * To be light-weight and copied by value.
+     * To allow logging statements to be left in source code.
+     * The logging is controlled at run-time based on a logging threshold.
+
+    It is advisable to check Journal::active(level) prior to formatting log
+    text.  Doing so sidesteps expensive text formatting when the results
+    will not be sent to the log.
+*/
+class Journal
+{
+public:
+     /** Severity level / threshold of a Journal message. */
+    using Severity = severities::Severity;
```

Comment:
Good idea.  I think I've already switched all of the users of `Severity` to reference `severities::Severity`